RealWordBE/Controllers/LoginController.cs

```diff
@@ -23,17 +24,47 @@ public LoginController(IUserService userService ,IMapper mapper)
         [HttpPost("users")]
         public async Task<ActionResult> RegisterAsync(RegisterOuterDto model)
         {
-            var userDto = model.userForRegisterDto;
+            var userDto = model.registerDto;
             var user = _mapper.Map<User>(userDto);
             var result = await _userService.RegisterAsync(user);
             if( result == "Success" )
             {
                 var response = _mapper.Map<UserResponseDto>(user);
-                //TO DO  redirect to get  user is not that:(
+                //TO DO  redirect to get 
                 return Ok(response);
             }
             else
-                return Ok(result);
+                return BadRequest(new Error
+                {
+                    Tittle = "Bad Request" ,
+                    Status = "400" ,
+                    ErrorMessage = result
+                });
+        }
```

Comment:
Bad request :is something that has to do with the submitted request from the client before it is even processed by the server